ibis/backends/tests/test_map.py

```diff
@@ -26,6 +28,34 @@
 ]
 
 
+@pytest.mark.notyet("clickhouse", reason="nested types can't be NULL")
+@pytest.mark.notimpl(["pandas", "dask"], reason="TypeError: iteration over a 0-d array")
+@pytest.mark.parametrize(
+    "kv",
+    [
+        param((["a", "b"], None), id="null_values"),
+        param((None, ["c", "d"]), id="null_keys"),
+        param((None, None), id="null_both"),
+    ],
+)
+def test_map_nulls(con, kv):
+    k, v = kv
+    k = ibis.literal(k, type="array<string>")
+    v = ibis.literal(v, type="array<string>")
+    m = ibis.map(k, v)
+    assert con.execute(m) is None
+
+
+@pytest.mark.xfail(
```

Comment:
Move this test to `ibis/backends/tests/test_client.py` or if there's another file that makes sense, then there.